python/cross_service/resilient_service/README.md

```diff
@@ -0,0 +1,183 @@
+
+## Scenario
+
+### Prerequisites
+
+To ensure that all of the components used in this example can access one another you must
+create them all in the same VPC. A VPC is a logically isolated network that is provided by
+[Amazon Virtual Private Cloud (Amazon VPC)](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).
+You can use either the default VPC that is included with your account or create your own.
+
+To ensure that you can access the load balancer endpoint, you must allow inbound traffic
+on port 80 from your computer's IP address to your VPC. You can do this by adding 
+a rule to the default security group for your VPC that specifies your computer's IP address
+as a source.
+
+For general prerequisites, see the [README](../../README.md#prerequisites) in the `python` folder.
+
+### Instructions
+
+For general instructions to run the examples, see the
+[README](../../README.md#run-the-examples) in the `python` folder.
+
+Run this example by running the following command in the folder that contains this README:
+
+```
+python runner.py --action all
+```
+
+This starts an interactive scenario that walks you through several aspects of creating a 
+resilient web service and lets you send requests to the load balancer endpoint and verify
+instance health along the way.
+
+#### Build and Manage a Resilient Service
+
+You can run the entire example by specifying `--action all` or run each of the sections 
+separately by specifying actions of `deploy`, `demo`, or `destroy`.
+
+##### Deploy resources
+
+Use the SDK for Python to create the following AWS resources: 
+
+1. A DynamoDB table that acts as a service that recommends books, movies, and songs.
+2. An instance profile and an associated role and policy that grants permission to
+   let instances access DynamoDB and Systems Manager.
+3. A launch template that specifies the instance profile and a startup script
+   that starts a Python web server on each instance.
+4. An Auto Scaling group that starts EC2 instances, one in each of three 
+   Availability Zones.
+5. An Application Load Balancer that handles HTTP requests to a single endpoint.
+6. A target group that connects the load balancer to instances in the Auto Scaling group.
+7. A listener that is added to the load balancer and forwards requests to the target group.  
+
+##### Demonstrate resiliency
+
+This part of the example demonstrates resiliency by simulating several kinds of failures.
+It uses Systems Manager parameters to update how the web server responds to requests and
+health checks to show how to make your web server more resilient to failure.
+
+Along with recommendations returned by the DynamoDB table, the web service includes the
+instance's ID and Availability Zone so you can see how the load balancer distributes
+requests among the instances in the Auto Scaling group.
+
+The scenario takes the following steps:  
+
+1. Sends requests to the endpoint to get recommendations and verify that instances 
```

Comment:
This is only a suggestion, because what you have here is pretty clear. But would it be helpful to prefix each of these steps with a sort of summary/title, to help clarify what is happening. So, for example:
1. **All Instances Healthy**: Sends requests...
2. **All Instances Fail with Error**: 
3. **All Instances Fail with Static Response**: .....

Again, only an idea, see what you think or ignore, your choice.

Reply:
Good idea, I had done something like this for the blog post, so I added that here as well!